### PR TITLE
Reformat into ES6 and a more standard style

### PR DESCRIPTION
**Pull request by snood1205**
I generally just modernized the code by taking advantage of the fact that node uses ES6. This allowed to make the code more functional and on the whole easier to read. No breaking changes or any significant changes in functionality.

### CODE REVIEW

File: care.js [hunk 1]
```diff
@@ -1,180 +1,167 @@
...
-var parrotSay = require('parrotsay-api');
-var weather = require('weather-js');
-
-var screen = blessed.screen(
-    {fullUnicode: true, // emoji or bust
-     smartCSR: true,
-     autoPadding: true,
-     title: '✨💖 tiny care terminal 💖✨'
-    });
+const config = require(__dirname + '/config.js')
```
Comment (notwaldorf) [inline care.js]: Sorry, but I'm really keen on semicolons. Can you add them back? (here and everywhere)

File: care.js [hunk 2]
```diff
@@ -1,180 +1,167 @@
...
-        var forecast = json.forecast[i];
-        if (forecast.day === currentDay) {
-          var skytextforecast = forecast.skytextday.toLowerCase();
-          forecastString = `Today, it will be ${skytextforecast} with the forecasted high of ${forecast.high}°${degreetype} and a low of ${forecast.low}°${degreetype}.`;
-        }
-      }
-      weatherBox.content = `In ${json.location.name} it's ${json.current.temperature}°${degreetype} and ${skytext} right now. ${forecastString}`;
+      const skytext = json.current.skytext.toLowerCase()
+      const currentDay = json.current.day
+      const skytextForecast = json.find((forceast) => forecast.day === currentDay).skytextday.toLowerCase()
```
Comment (notwaldorf) [inline care.js]: forecast typo

Comment (notwaldorf) [inline care.js]: also wow, what is this `.find` sorcery? TIL.

File: care.js [hunk 3]
```diff
@@ -1,180 +1,167 @@
...
+      const forecastString = `Today, it will be ${skytextforecast} with the forecasted high of ${forecast.high}°${degreetype} and a low of ${forecast.low}°${degreetype}.`
```
Comment (notwaldorf) [inline care.js]: it was `skytextForecast` before (lost the capital F)

File: care.js [hunk 4]
```diff
@@ -1,180 +1,167 @@
...
+const parrotBox = grid.set(6, 6, 6, 6, blessed.box, makeScrollBox(''))
+
+const tweetBoxes = {}
+tweetBoxes[config.twitter[1]] = grid.set(2, 8, 2, 4, blessed.box, makeBox(' 💖 '))
+tweetBoxes[config.twitter[2]] = grid.set(4, 8, 2, 4, blessed.box, makeBox(' 💬 '))
+
+tick()
+setInterval(tick, 1000 * 60 * config.updateInterval)
+
+function tick () {
```
Comment (notwaldorf) [inline care.js]: No space between the function name and the (), pls

File: care.js [hunk 5]
```diff
@@ -1,180 +1,167 @@
...
+  doTheWeather()
+  doTheTweets()
+  doTheCodes()
 }
 
-function doTheWeather() {
-  weather.find({search: config.weather, degreeType: config.celsius ? 'C' : 'F'}, function(err, result) {
+function doTheWeather () {
```
Comment (notwaldorf) [inline care.js]: no space between function name and ()

File: care.js [hunk 6]
```diff
@@ -1,180 +1,167 @@
...
-      weatherBox.content = 'Having trouble fetching the weather for you :(';
+      weatherBox.content = 'Having trouble fetching the weather for you :('
     }
-  });
+  })
 }
 
-function doTheTweets() {
-  for (var which in config.twitter) {
+function doTheTweets () {
```
Comment (notwaldorf) [inline care.js]: no space between function name and ()

Comment (notwaldorf) [inline care.js]: (here and everywhere)

File: twitterbot.js [hunk 7]
```diff
@@ -1,50 +1,43 @@
...
+const Twit = require('twit')
+const config = require(__dirname + '/config.js')
+const scraperjs = require('scraperjs')
 
-var T = new Twit({
-  consumer_key:        config.keys.consumer_key,
-  consumer_secret:     config.keys.consumer_secret,
-  access_token:        config.keys.access_token,
+const T = new Twit({
+  consumer_key: config.keys.consumer_key,
```
Comment (notwaldorf) [inline twitterbot.js]: please re-indent these values like before -- the alignment was nice

File: config.js [hunk 8]
```diff
@@ -1,4 +1,4 @@
-var config = {
+const config = {
```
Comment (notwaldorf) [inline config.js]: Would you mind leaving this as a `var` please? I muck around with it for testing.

File: config.js [hunk 9]
```diff
@@ -10,21 +10,21 @@ var config = {
   // Directories in which to run git-standup on for a list of your recent commits.
   repos: (process.env.TTC_REPOS || '~/Code').replace(/,/g, ' '),
 
-  // Where to check the weather for.
-  // It's using weather.service.msn.com behind the curtains.
+// Where to check the weather for.
```
Comment (notwaldorf) [inline config.js]: Wrong indent

File: config.js [hunk 10]
```diff
@@ -10,21 +10,21 @@ var config = {
...
   celsius: (process.env.TTC_CELSIUS || 'true') === 'true',
-
   updateInterval: parseFloat(process.env.TTC_UPDATE_INTERVAL) || 20,
+  // Set to false if you're an imperial savage. <3
 
   keys: {
-    consumer_key:        process.env.TTC_CONSUMER_KEY || process.env.CONSUMER_KEY || 'none',
-    consumer_secret:     process.env.TTC_CONSUMER_SECRET || process.env.CONSUMER_SECRET || 'none',
-    access_token:        process.env.TTC_ACCESS_TOKEN || process.env.ACCESS_TOKEN || 'none',
+    consumer_key: process.env.TTC_CONSUMER_KEY || process.env.CONSUMER_KEY || 'none',
```
Comment (notwaldorf) [inline config.js]: please re-align the values as before

File: config.js [hunk 11]
```diff
@@ -15,9 +15,9 @@ var config = {
   weather: process.env.TTC_WEATHER || 'San Francisco',
 
   // Set to false if you're an imperial savage. <3
-  celsius: (process.env.TTC_CELSIUS || 'true') === 'true',
-
+  celsius:        (process.env.TTC_CELSIUS || 'true') === 'true',
   updateInterval: parseFloat(process.env.TTC_UPDATE_INTERVAL) || 20,
+  // Set to false if you're an imperial savage. <3
```
Comment (notwaldorf) [inline config.js]: Oops, i think this needs to get moved back up. It'

File: care.js [hunk 12]
```diff
@@ -49,70 +46,64 @@ function tick() {
...
-          var skytextforecast = forecast.skytextday.toLowerCase();
-          forecastString = `Today, it will be ${skytextforecast} with the forecasted high of ${forecast.high}°${degreetype} and a low of ${forecast.low}°${degreetype}.`;
-        }
-      }
-      weatherBox.content = `In ${json.location.name} it's ${json.current.temperature}°${degreetype} and ${skytext} right now. ${forecastString}`;
+      const skytext         = json.current.skytext.toLowerCase();
+      const currentDay      = json.current.day;
+      const skytextForecast = json.find((forecast) => forecast.day === currentDay).skytextday.toLowerCase();
+      const forecastString  = `Today, it will be ${skytextForecast} with the forecasted high of ${forecast.high}°${degreetype} and a low of ${forecast.low}°${degreetype}.`;
+      const degreetype      = json.location.degreetype;
```
Comment (notwaldorf) [inline care.js]: deegreetype is defined here but used above

File: config.js [hunk 13]
```diff
@@ -1,4 +1,4 @@
-var config = {
+let config = {
```
Comment (OmgImAlexis) [inline config.js]: Use const here?

Comment (snood1205) [inline config.js]: I was requested by @notwaldorf to use `let` instead of `const`.

Comment (notwaldorf) [inline config.js]: yeah, is nice for testing 🙏

**Comment by notwaldorf** [comment]
Closing, since this PR has unadressed requested changes and desperately needs a rebase

**Pull request closed without merging**